test/test/jfr/JfrTests.java

```diff
@@ -88,10 +88,10 @@ public void parseMultiModeRecording(TestProcess p) throws Exception {
      * @param p The test process to profile with.
      * @throws Exception Any exception thrown during profiling JFR output parsing.
      */
-    @Test(mainClass = JfrMultiModeProfiling.class, agentArgs = "start,all,file=%f.jfr", os = Os.LINUX)
-    @Test(mainClass = JfrMultiModeProfiling.class, agentArgs = "start,all,alloc=100,file=%f.jfr", os = Os.LINUX)
-    @Test(mainClass = JfrMultiModeProfiling.class, agentArgs = "start,all,file=%f.jfr", os = Os.MACOS, arch = Arch.ARM64)
-    @Test(mainClass = JfrMultiModeProfiling.class, agentArgs = "start,all,lock=10ms,file=%f.jfr", os = Os.MACOS, arch = Arch.ARM64)
+    @Test(mainClass = JfrMultiModeProfiling.class, agentArgs = "start,all,file=%f.jfr", os = Os.LINUX, jvmVer = {11, Integer.MAX_VALUE})
```

Comment:
Why 11+? Is it because of `live` profiling not supported on JDK 8?
I think `--all` should still work on JDK 8, although without live object profiling.

Reply:
Yes, failure here: https://github.com/async-profiler/async-profiler/actions/runs/15116002786/job/42486783966#step:10:113

I'll fix the failure in this PR

Reply:
d3784df4b887503ef4baae1e5d0ae2d1cd4a336d